frontends/open-discussions/src/pages/CourseIndexPage.js

```diff
@@ -103,13 +103,13 @@ export default function CourseIndexPage({ history }: Props) {
           }}
         >
           <ResponsiveWrapper onlyOn={[TABLET, DESKTOP]}>
-            <div className="link-wrapper">
-              <Link className="link-button" to={COURSE_SEARCH_URL}>
+            <div className="link-wrapper d-flex justify-content-end mt-3">
+              <Link className="link-button me-0" to={COURSE_SEARCH_URL}>
```

Comment:
Why these new classes?

Previously, CourseSearchbox applied `course-searchbox` class, and that class affected the style of the the link wrappers and link buttons inside of it. 

The new styling for Searchbox does not affect the children in this way. Instead, children need to be responsible for styling themselves.

This was the only case that is affected, and the child styling is easy to get with a few of bootstrap's utility classes.